### ISSUE TITLE
Full Screen doesn't remove the toobars and borders

### ISSUE THREAD

**Issue by KorigamiK** [zen-browser/desktop#3601]
### Captchas

- [X] I have read the instructions.
- [X] I have searched existing issues and avoided creating duplicates.
- [X] I am not filing an enhancement request.

### What happened?

Full screen just makes the content go full screen, it doesn't collapse the toolbars and stuff.
A keybind would also be very nice

### Reproducible?

- [X] I have checked that this issue cannot be reproduced on Mozilla Firefox.

### Version

Mozilla Zen 1.0.2-b.0

### What platform are you seeing the problem on?

Linux

**Comment by ekorchmar** [comment]
Duplicate of #614

**Comment by androidghosh** [comment]
Issue is still present. Please fix

**Comment by JSGRD22** [comment]
This is intentional. You can hide browser elements by activating compact mode then going into fullscreen.

**Comment by JSGRD22** [comment]
This is intentional. You can hide browser elements by activating compact mode then going into fullscreen.

**Comment by KorigamiK** [comment]
> This is intentional. You can hide browser elements by activating compact mode then going into fullscreen.

Yeah but it's overriding the default behaviour. Would be great if we can have a toggle for this

**Comment by ekorchmar** [comment]
This is a clear regression from Firefox behavior: all code that enables this behavior on Firefox is still here, on Zen; only reason it is not working is because Zen has renamed/reworked original Firefox UI elements into new ones, but does not apply old behavior to them.

Hiding UI elements on entering full screen is a reasonable expectation for behavior, and is implemented by every desktop browser out there. Please consider reopening #614, as there are more documented details to this issue.

Also, compact mode still shows UI elements (rounded frame), so it does not solve the issue.

**Comment by jnki** [comment]
please add real fullscreen like every other browser.
F11 + compact mode is not fullscreen, as the border is still visible.

**Comment by KorigamiK** [comment]
Yup this issue is pretty much solved for me
